src/config.js

```diff
@@ -1,15 +1,15 @@
 const fs = require('fs');
 const path = require('path');
 
-const { setToken } = require('./github');
-const { getPackageDetails } = require('./utils');
+const GITHUB = require('./github');
+const UTILS = require('./utils');
 
-const { package: PACKAGE, author: PACKAGE_AUTHOR } = getPackageDetails();
+const { package: PACKAGE, author: PACKAGE_AUTHOR } = UTILS.getPackageDetails();
 const HOME_DIR = require('os').homedir();
 const CONFIG_DIR = getConfigDir(HOME_DIR);
 const CONFIG_FILE = path.join(CONFIG_DIR, 'auth.json');
 
-async function saveConfig(token) {
+async function save(token) {
```

Comment:
This doesn't need `async` since there is no `await` in the function, you're just using the `*Sync` version of the FS functions.